support-frontend/assets/helpers/page/commonReducer.ts

```diff
@@ -97,7 +97,10 @@ function createCommonReducer(initialState: CommonState) {
 					amounts:
 						action.useLocalAmounts &&
 						state.internationalisation.localCurrencyCountry
-							? state.internationalisation.localCurrencyCountry.amounts
+							? {
+									...state.defaultAmounts,
+									...state.internationalisation.localCurrencyCountry.amounts,
+							  }
```

Comment:
This will need moving over into the new Redux Toolkit reducer- I think it should be pretty straightforward as this isn't a big change, but give me a shout if you need any help.

Reply:
Yeah it was super straightforward! Gosh that new slice is good isn't it?